src/verus.pest

```diff
@@ -1733,6 +1735,10 @@ ensures_clause = {
     ensures_str ~ comma_delimited_exprs_for_verus_clauses?
 }
 
+returns_clause = {
+    returns_str ~ comma_delimited_exprs_for_verus_clauses?
```

Comment:
Should this be more restrictive?  This allows `returns a, b, 5,` for example.

Reply:
Agreed, probably could be `returns_str ~ expr_no_struct ~ ","?` or similar?